.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-./node_modules/lint-staged/bin/lint-staged.js
+npx lint-staged
```

Comment:
Bear in mind when using npx to run the command it will always assume the latest unless you supply a version after the command. So will ignore your package.json version

Reply:
Yep or have that `npm run` script to run the locally installed version

Reply:
i've reverted it for now